msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-meeting-stage.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Build tabs for meeting
```

Comment:
If this is "Build tabs for meeting", should we rename the filename of the MD file? this is a new file so not sure why we wouldn't just have it match.

Reply:
agreed, thanks for catching it @v-sdhakshina please update file name